stencil-library/docs.json

```diff
@@ -10,7 +10,7 @@
       "filePath": "./src/components/bank-account-form/bank-account-form.tsx",
       "encapsulation": "none",
       "tag": "justifi-bank-account-form",
-      "readme": "# justifi-bank-account-form\n\n\n",
+      "readme": "# tag: <justifi-bank-account-form />\n\n\n",
```

Comment:
Is it possible to use markdown in this? Just curious. I think IF we can we should do `<justifi-bank-account-form />` (<- like apply the code fencing syntax like this) instead of "tag:". Just a thought, not important 😄

Reply:
It could definitely be an option, let me try it out